Cargo.toml

```diff
@@ -48,3 +48,8 @@ incremental = false
 lto = 'fat'
 opt-level = 3 # <-
 overflow-checks = false # <-
+
+# [patch.crates-io]
```

Comment:
nit: could we add a small comment like 
`# this serves as a configuration template if you want to use the git version of defmt instead of the one supplied by crates.io. Check the app-template README for more information`
so that people stumbling across this before they've finished the README know what it's for?

Reply:
absolutely, this should have a description comment

Reply:
added a description